packages/wagtail/modules/services/Wagtail/PagesAPI/index.ts

```
@@ -0,0 +1,25 @@
+import { WagtailAPI } from '..'
+import { IPage } from './types'
+
+export class PagesAPI extends WagtailAPI {
+  static namespace = `base/pages`
+
+  static getPageByPath(path: string, extraParams: any = {}): Promise<IPage> {
+    return this.axios.get(`${this.baseURL}/${this.namespace}/path/`, {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)